front/app/containers/ProjectsShowPage/timeline/PhaseReport/index.tsx

```diff
@@ -57,7 +57,7 @@ const PhaseReport = ({ reportId, phaseId }: Props) => {
           <Box
             w="100%"
             display="flex"
-            alignItems="center"
+            alignItems="flex-start"
```

Comment:
Also unrelated- in the designs, the report is aligned to the left. Forgot that I still had to do this